From 1420cae67edb1a7e490bb57f9f50074edd4dea74 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Mon, 27 May 2013 17:16:35 +0300
Subject: [PATCH 2/2] dt: bone-common: Add interrupt for PMIC

Add support for the PMIC interrupt, supports power-button presses.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 arch/arm/boot/dts/am335x-bone-common.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
index aa95810..c450d0b 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -431,6 +431,9 @@
 &tps {
 	ti,pmic-shutdown-controller;
 
+	interrupt-parent = <&intc>;
+	interrupts = <7>;	/* NNMI */
+
 	regulators {
 		dcdc1_reg: regulator@0 {
 			regulator-always-on;
@@ -472,6 +475,7 @@
 			regulator-always-on;
 		};
 	};
+
 };
 
 &cpsw_emac0 {
-- 
1.7.9.5

